src/controllers/browser.ts

```diff
@@ -10,7 +10,7 @@ export async function initWhatsapp(
   const browser = await initBrowser(session, headless, devtools);
   const waPage = await getWhatsappPage(browser);
   await waPage.setUserAgent(
-    'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3312.0 Safari/537.36'
+    'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.0 Safari/537.36'
```

Comment:
Don't you think it's better to use the UA of a more popular "real user" OS?

Reply:
I have use my pc user agent :) If you want I can revert that part to the previous one

Reply:
There seems to be another problem, I have to remove the session folder after each restart or the userAgent is not correctly working and the qr code is not shown, instead I get a page that tells me to update chrome. To fix this I'm tring to use some other flags